buildrunner/steprunner/tasks/build.py

```diff
@@ -210,16 +210,27 @@ def run(self, context):
             docker_registry=self.step_runner.build_runner.global_config.get_docker_registry(),
         )
         try:
-            exit_code = builder.build(
-                console=self.step_runner.log,
-                nocache=self.nocache,
-                cache_from=self.cache_from,
-                pull=self.pull,
-                buildargs=self.buildargs,
-                platform=self.platform
-            )
-            if exit_code != 0 or not builder.image:
-                raise BuildRunnerProcessingError('Error building image')
+            if isinstance(self.platform, list):
+                built_images = self.step_runner.multi_platform.build(platforms=self.platform,
+                                                                     path=self.path,
+                                                                     file=self.config.get('dockerfile'),
+                                                                     name=self.get_unique_build_name(),)
+
+                assert len(built_images) == len(self.platform), \
+                    f'Number of built images ({len(built_images)}) does not match ' \
+                    'the number of platforms ({len(self.platform)})'
```

Comment:
when splitting a string you need the `f` prefix on every formatted string

Reply:
done